### PR TITLE
Update "How to research a technology"

### PR DESCRIPTION
**Pull request by yarusome**
### Description

This PR tweaks some phrasings in [How to research a technology](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Howto/Research_technology).

### Motivation

This blocks the l10n of this page.

### Related issues and pull requests

### CODE REVIEW

File: files/en-us/mdn/writing_guidelines/howto/research_technology/index.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Useful resources for writing any documentation include:
...
-3. The latest modern web browsers: These should be experimental/alpha builds such as [Firefox Nightly](https://www.mozilla.org/en-US/firefox/channel/desktop/#nightly), [Chrome Canary](https://www.google.com/intl/en/chrome/canary/), or [Safari Technology Preview](https://webkit.org/downloads/) that are more likely to support the features you are documenting. This is especially pertinent if you are documenting a feature that is "upcoming".
+3. The latest modern web browsers: These should be experimental/alpha builds such as [Firefox Nightly](https://www.mozilla.org/en-US/firefox/channel/desktop/#nightly), [Chrome Canary](https://www.google.com/intl/en/chrome/canary/), or [WebKit Build](https://webkit.org/downloads/#nightly) that are more likely to support the features you are documenting. This is especially pertinent if you are documenting a feature that is "upcoming".
```
Comment (hamishwillee) [inline files/en-us/mdn/writing_guidelines/howto/research_technology/index.md:28]: Isn't the Safari preview the right one? BCD doesn't report on WebKit but Safari.

Further, https://webkit.org/downloads/ provides both the safari preview and webkit.

Comment (yarusome) [reply files/en-us/mdn/writing_guidelines/howto/research_technology/index.md]: I have reverted this commit, but it seems that either Safari TP or Webkit Build has some inconsistency here as Safari TP is released on a monthly basis, which corresponds more intuitively to Firefox Beta/Dev. :thinking:

File: files/en-us/mdn/writing_guidelines/howto/research_technology/index.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ page-type: mdn-writing-guide
 
 This article gives you some handy information about approaching how to document technologies.
 
-## Do the prep work
+## Doing the prep work
```
[the remark this answers is not in the document]
Comment (hamishwillee) [reply files/en-us/mdn/writing_guidelines/howto/research_technology/index.md:11]: Me either, but this is no worse.

File: files/en-us/mdn/writing_guidelines/howto/research_technology/index.md [hunk 3]
```diff
@@ -19,52 +19,48 @@ It is assumed that before reading this guide, you have a reasonable knowledge of
...
 
-It's possible that the sidebar of the pages you write will also need to be defined or updated. To find out if this is needed and how to do it, [check out our sidebar guide](/en-US/docs/MDN/Writing_guidelines/Howto/Write_an_API_reference/Sidebars).
+It's possible that the sidebar of the pages you write will also need to be defined or updated. To find out if this is needed and how to do it, check out the [sidebar guide](/en-US/docs/MDN/Writing_guidelines/Howto/Write_an_API_reference/Sidebars).
 
 ### Code examples
 
 Some of the code examples for MDN Web Docs are held in separate repositories. Most notably, these are the interactive examples that appear in the "Try it" section in the reference pages and the larger demo code needed for guides. If you do need to add to or amend one of these repositories, it's a good idea to make a note of it in your list.
 
-This [article](/en-US/docs/MDN/Writing_guidelines/Page_structures/Code_examples) describes the different types of code examples we use on MDN Web Docs.
+[This article](/en-US/docs/MDN/Writing_guidelines/Page_structures/Code_examples) describes the different types of code examples we use on MDN Web Docs.
```
Comment (hamishwillee) [inline files/en-us/mdn/writing_guidelines/howto/research_technology/index.md:63]:
```suggestion
The [Code examples](/en-US/docs/MDN/Writing_guidelines/Page_structures/Code_examples) article describes the different types of code examples we use on MDN Web Docs.
```

**Review by hamishwillee — APPROVED** [review]
Thank you.

File: files/en-us/mdn/writing_guidelines/howto/research_technology/index.md (hunk 1 above)
Comment (hamishwillee) [reply files/en-us/mdn/writing_guidelines/howto/research_technology/index.md]: I think the addition of safari TP is good - that's the test target in browser compatibility target for preview.

**Pull request merged**